src/Tasks/Microsoft.NET.Build.Tasks/PrepareForReadyToRunCompilation.cs

```diff
@@ -170,7 +158,7 @@ protected override void ExecuteCore()
                         outputPDBImageRelativePath = Path.ChangeExtension(outputR2RImageRelativePath, "ni.pdb");
                         crossgen1CreatePDBCommand = $"/CreatePDB \"{Path.GetDirectoryName(outputPDBImage)}\"";
                     }
-                    else if (IsTargetLinux)
+                    else
```

Comment:
I do not think there is any tooling on Windows that is capable of consuming the perfmap files.

It probably does not make sense to produce the perfmap file when we are on Windows and we do not have good diasymreader.

Reply:
OK, sounds reasonable. I initially thought it might be useful for various experimental purposes but you're probably right that it's better to keep the logic simple and consistent. I have modified my change based on your feedback in the 2nd commit.